remo/profiles/tasks.py

```diff
@@ -108,3 +114,21 @@ def set_unavailability_flag():
     (UserStatus.objects.filter(start_date__range=[get_date(-1), get_date()],
                                is_unavailable=False)
                        .update(is_unavailable=True))
+
+
+@periodic_task(run_every=timedelta(hours=24))
+def resolve_nomination_action_items():
+    """Resolve action items.
+
+    Resolve all the action items relevant to nomination reminders after the
+    10th day of each month.
+    """
+
+    if now().date().day == NOMINATION_END_DAY:
+        mentors = UserProfile.objects.filter(user__groups__name='Mentor')
+        action_model = ContentType.objects.get_for_model(UserProfile)
+        # All the completed action items are always resolved
+        items = (ActionItem.objects.filter(content_type=action_model,
```

Comment:
Would it make sense here instead of filtering all ``UserProfile`` related action items to filter only those related to ROTM?